src/Stripe.Tests.XUnit/Stripe.Tests.XUnit.csproj

```diff
@@ -8,7 +8,7 @@
     <RuntimeIdentifiers>win10-x64;win81-x64;osx.10.12-x64</RuntimeIdentifiers>
     <RuntimeFrameworkVersion>1.1.1</RuntimeFrameworkVersion>
     <PackageTargetFallback>$(PackageTargetFallback);dotnet5.6;portable-net45+win8</PackageTargetFallback>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <NetStandardImplicitPackageVersion>1.2</NetStandardImplicitPackageVersion>
```

Comment:
The cipher suites available may be an issue? I believe TLS 1.2 is supported all around here, but it has to be specified like [this](https://stackoverflow.com/questions/39560249/servicepointmanager-in-asp-net-core) or  the `ServicePointManager` that is being used now. Worth more investigation.